app/preload/ledger/ledgerObserverImpl.ts

```diff
@@ -37,221 +90,200 @@ export default class LedgerObserverImpl implements LedgerObserver {
         return this.concordiumClient;
     }
 
-    private pollingInterval: NodeJS.Timeout | null = null;
+    private pollingInterval: ReturnType<typeof setInterval> | null = null;
 
-    /**
-     * Polls the Ledger device every 2 seconds to check if the Concordium app is open.
-     * If the app is found, it emits an action to the main window indicating whether
-     * the app is outdated or connected. It stops polling once the app is detected.
-     *
-     * @param mainWindow - The main window's EventEmitter to emit actions to the window.
-     * @param deviceName - An optional name of the device to include in the emitted action.
-     *
-     * The polling interval is cleared when the Concordium app is detected or if any transient errors occur.
-     */
```

Comment:
Can we keep the comments please? I don't think it makes a lot of sense to remove it as it helps people get an overview. If the functionality changed, I would rather have us update the comment than remove it